CalWebViewApp/bin/make/ipa-cal.sh

```diff
@@ -127,7 +127,7 @@ fi
 cat >"${XTC_DIR}/Gemfile" <<EOF
 source "https://rubygems.org"
 
-gem "calabash-cucumber", "2.99.0"
+gem "calabash-cucumber"
```

Comment:
This must be 2.99.0.  It is the last allowed version in Test Cloud.

Reply:
Latest version available at the moment is `0.21.10`.